test/DataFactory.js

```diff
@@ -435,6 +435,52 @@
                     ["Label1", 45.777062, -108.549835, "red-dot.png", 4525, 423545, 24354, 54, 25523345, 2455, 245435]
                 ],
                 columns: ["Label", "Lat", "Long", "Pin", "Forth Column", "Fifth Column", "sixth Column", "Seventh Column", "eighth Column", "Nineth Column", "Tenth Column"]
+            },
+            widget: {
+                data: [
+                    ["Label1", 37.665074, -122.384375, "green-dot.png", 4525, 4243545, 24354, 54, 2552345, 2455, 245435],
+                    ["Label2", 32.690680, -117.178540, new Input().name("checkbox-test").label("Checkbox Test").type("checkbox").value(true), 4525, 423545, 24354, 354, 2552345, 2455, 245435],
+                    ["Label3", 39.709455, -104.969859, new Table().fixedSize(true).fixedHeader(false).columns( ["Subject","Year 1", "Year 2", "Year 3"]).data([["Math",76,65,87],["History",65,87,67],["Science",88,91,78]]), 4525, 423545, 24354, 524, 2552345, 2455, 245435],
+                    ["Label4", 41.244123, -95.961610, new Column().size({width: 200, height: 200}).columns( ["Subject","Year 1", "Year 2", "Year 3"]).data([["Math",76,65,87],["History",65,87,67],["Science",88,91,78]]), 4525, 423545, 24354, 564, 2552345, 2455, 245435],
```

Comment:
Note the column needs to have a size assigned.  The table, in this instance, you want to have its natural size, whatever that may be.